src/main/java/core/mate/academy/model/ExcavatorProducer.java

```diff
@@ -0,0 +1,18 @@
+package core.mate.academy.model;
+
+import java.util.List;
+import core.mate.academy.service.MachineProducer;
+
+public class ExcavatorProducer implements MachineProducer<Excavator> {
+
+    @Override
+    public List<Excavator> get() {
+        Excavator jsb = new Excavator("Name : jcb 3trx", "Color : yellow",
+                2000,
+                400);
+        Excavator trx = new Excavator("Name : trx 800pro", "Color : red",
```

Comment:
Consider removing the 'Name :' and 'Color :' prefixes from the excavator names and colors for consistency and clarity. The constructor parameters should directly represent the name and color values without additional text.